src/main.rs

```diff
@@ -56,34 +59,73 @@ fn main() {
                 // Can this ever happen?
                 Ok(())
             } else {
-                println!("Fuzzing found errors!");
+                if let Some(ref mut terminal) = term_stdout {
+                    let _ = terminal.fg(term::color::YELLOW);
+                    println!("Fuzzing found errors!");
+                    let _ = terminal.reset();
+                } else {
+                    println!("Fuzzing found errors!");
+                }
```

Comment:
yes we can. I suggest to move it to a separate module, like `utils`, where we can add other logic like the one from `term`